scripts/docs_update_release.py

```diff
@@ -0,0 +1,224 @@
+    except FileNotFoundError as e:
+        print(f"Error opening file {INDEX_PATH} - {e}")
+        raise SystemExit(1)
+
+    # find every element with the class "download-button"
+    elements = []
+    for i in range(0, len(index_lines)):
+        if index_lines[i].strip() == '<div class="download-button">':
+            elements.append([i, index_lines[i:i+10]])
+            i += 10 # skip the next 10 lines (the length of the element with the class "download-button")
+
+    # choose the elements that contain the installer type
+    selected_elements = []
+    for element in elements:
+        if any(f'{installer_type}' in element_line for element_line in element[1]):
+            selected_elements.append(element)
+
+    print(f"Found {len(selected_elements)} elements with the installer type {installer_type}")
+
+
+    def _replace_installer_button(element_to_replace:list) -> str:
+        """Replace the first occurrence of the installer button with the new one"""
+        element_to_replace = ''.join(element_to_replace[1])
+        print(f"This element will be replaced:\n{element_to_replace}")
+
+        index_data = ''.join(index_lines)
+        return index_data.replace(element_to_replace, _resolve_installer_type())
+
+
+    # replace the first occurrence of the offline installer button
+    if installer_type == 'offline':
+        element_to_replace = None
+        for selected_element in selected_elements:
+            for element_line in selected_element[1]:
+                if f'{idf_version[0]}.{idf_version[1]}' in element_line:
+                    element_to_replace = selected_element
+                    break
+        if element_to_replace:
+            new_index_data = _replace_installer_button(element_to_replace)
+        else:   # add new installer button to the top of the offline installer buttons
+            first_occurrence = selected_elements[0][0]
+
+            print(f"First occurrence on line {first_occurrence} - adding new installer button here")
+            index_data = index_lines[0:first_occurrence-1] + list(_resolve_installer_type()+'\n') + index_lines[first_occurrence:]
+            new_index_data = ''.join(index_data)
+    else:   # replace the first occurrence of the other installer type button
+        new_index_data = _replace_installer_button(selected_elements[0])
+
+
+    try:
+        with open(path.abspath(INDEX_PATH), "w") as index_file:
+            index_file.write(new_index_data)
+    except FileNotFoundError as e:
+        print(f"Error writing file {INDEX_PATH} - {e}")
+        raise SystemExit(1)
+
+
+
+def update_releases_json():
+    """Update the releases.json file with the new release of the installer"""
+    try:
+        with open(path.abspath(RELEASES_JSON_PATH), "r") as releases_file:
+            releases_data = releases_file.read()
+    except FileNotFoundError as e:
+        print(f"Error opening file {RELEASES_JSON_PATH} - {e}")
+        raise SystemExit(1)
```

Comment:
Consider using the text in SystemExit, instead of printing it. In that case the message will be printed to stderr.